app/addons/documents/components/results-toolbar.js

```diff
@@ -83,5 +97,7 @@ ResultsToolBar.propTypes = {
   isLoading: PropTypes.bool.isRequired,
   hasResults: PropTypes.bool.isRequired,
   isListDeletable: PropTypes.bool,
-  partitionKey: PropTypes.string
+  partitionKey: PropTypes.string,
```

Comment:
Unrelated?

Reply:
Yes, the `partitionKey` was introduced in another PR